src/pyclvm/instance/start.py

```diff
@@ -1,17 +1,35 @@
-from typing import Optional, Tuple
+from functools import partial
+from typing import Any, Callable, Optional, Tuple
 
 from boto3.session import Session
 
 from ._mapping import Instance
 from ._process import process_instances
 
 
-def _start_instance(instance_name: str, instance: Instance) -> None:
-    if instance.state.Name == "stopped":
-        print(f"Starting {instance_name} ...")
-        instance.start()
-        instance.wait_until_running()
+def _start_instance(
+    instance_name: str, instance: Instance
+) -> Optional[Callable[..., Any]]:
+    _state = instance.state.Name
```

Comment:
How about removing redundant variables? The same for stop action.
Something like:
```
return {
        "running": partial(_is_running, instance_name),
        "stopped": partial(_is_stopped_or_terminated, instance_name, instance),
        "terminated": partial(_is_stopped_or_terminated, instance_name, instance),
    }[instance.state.Name]
```